addons/payment_authorize/views/payment_views.xml

```diff
@@ -30,7 +30,7 @@
             <xpath expr="//field[@name='country_ids']" position="after">
                 <label for="authorize_currency_id" string="Currency" attrs="{'invisible': [('provider', '!=', 'authorize')]}"/>
                 <div attrs="{'invisible': [('provider', '!=', 'authorize')]}">
-                    <field name="authorize_currency_id"/>
+                    <field name="authorize_currency_id" required="1"/>
```

Comment:
That's not going to work for two reasons:
- Using `required="1"` rather than `attrs="{'invisible': ...}"` will prevent saving any other acquirer form.
- Making the field required without any other adaptation to the view will prevent using the dedicated buttons to automatically set the `authorize_currency_id` and `authorize_client_key` fields.